### PR TITLE
Add method to return the significance of a track inside a Det

### PR DESCRIPTION
**Pull request by VinInn**
With this PR we add to ```Bounds```  a method that returns the significance of a track inside a Det
(negative if inside, positive if outside)
We add this information to "MIssingHIts" in place of the "estimator".

The idea is to later use this information to try to identify tracks that miss a Det because they pass between the gap among two of them: this has been seen occurring in particular in the Inner Pixel for both Phase1 and Phase2.

This PR just adds the capability, does not make any use of it.
No regression expected. It does recompile the universe.
Any future use of this information is expected to create regression: so we prefer to start from an IB
already including this basic mods that trigger major recompilation.

### CODE REVIEW

**Comment by VinInn** [comment]
@cmsbuild, please test

**Comment by civanch** [comment]
+1

File: DataFormats/GeometrySurface/interface/Bounds.h [hunk 1]
```diff
@@ -65,6 +65,8 @@ class Bounds {
     return inside( Local3DPoint(p.x(), p.y(), 0), err, scale);
   }
 
+  virtual float howMuchInside(const Local3DPoint&, const LocalError&) const;
```
Comment (slava77) [inline DataFormats/GeometrySurface/interface/Bounds.h]: significanceInside or similar seems more descriptive.

**Comment by VinInn** [comment]
Thanks @slava77 for the suggestion. edited in place

**Comment by VinInn** [comment]
@cmsbuild, please test

**Comment by slava77** [comment]
On 11/12/17 1:22 PM, Vincenzo Innocente wrote:
> Thanks @slava77 <https://github.com/slava77> for the suggestion. edited
> in place

Thank you.

BTW, what is the editing "in place"?
Is it from the web interactively? I was always wondering if that works.

**Comment by VinInn** [comment]
> On 12 Nov, 2017, at 2:02 PM, Slava Krutelyov <notifications@github.com> wrote:
> 
> BTW, what is the editing "in place"?
> Is it from the web interactively? I was always wondering if that works.
yes, I do it often for small changes and always for .md
 I was hoping to edit all files in one go, instead ended up in  editing them one by one.
Sorry for the commit spam!

v.

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/21147	c83121b
- significanceInside method for bound surfaces based on the input position and uncertainty. This information is saved with invalid hits in the track trajectory LayerMeasurements::addInvalidMeas; it does not affect monitored quantities.
- jenkins tests pass and comparisons show no differences

**Comment by davidlange6** [comment]
merge

**Pull request merged**